moped-editor/.env-cmdrc

```diff
@@ -107,6 +119,7 @@
     "REACT_APP_KNACK_DATA_TRACKER_PROJECT_ID_FIELD": "field_4133",
     "REACT_APP_KNACK_DATA_TRACKER_PROJECT_NAME_FIELD": "field_3857",
     "REACT_APP_KNACK_DATA_TRACKER_CURRENT_STATUS_FIELD": "field_4136",
-    "REACT_APP_KNACK_DATA_TRACKER_SIGNAL_CONNECTION": "field_3861"
+    "REACT_APP_KNACK_DATA_TRACKER_SIGNAL_CONNECTION": "field_3861",
+    "REACT_APP_KNACK_DATA_TRACKER_URL_BASE": "https://mobility.austin.gov/moped/projects/"
```

Comment:
I noticed this doesnt have the link_label nor moped_url env variable. Is that okay?

Reply:
Not OK! Thank you very much for catching these omissions! 🙏